agenta-backend/agenta_backend/migrations/mongo_to_postgres/migration.py

```diff
@@ -0,0 +1,548 @@
+        "updated_at": get_datetime(test_set.get("updated_at")),
+    }
+
+
+async def transform_evaluator_config(config):
+    app_uuid = await get_mapped_uuid("app_db", config["app"].id)
+    user_uuid = await get_mapped_uuid("users", config["user"].id)
+    config_uuid = generate_uuid()
+    await store_mapping("evaluators_configs", config["_id"], config_uuid)
+    return {
+        "id": config_uuid,
+        "app_id": app_uuid,
+        "user_id": user_uuid,
+        "name": config["name"],
+        "evaluator_key": config["evaluator_key"],
+        "settings_values": config["settings_values"],
+        "created_at": get_datetime(config.get("created_at")),
+        "updated_at": get_datetime(config.get("updated_at")),
+    }
+
+
+async def convert_human_evaluations_associated_variants(
+    variants, variants_revisions, evaluation_id
+):
+    """Convert variant and revision ObjectIds to UUIDs and structure them."""
+    associated_variants = []
+    assert len(variants) == len(
+        variants_revisions
+    ), "variants and variants_revisions must have the same length"
+
+    for variant_id, revision_id in zip(variants, variants_revisions):
+        variant_uuid = await get_mapped_uuid("app_variants", variant_id)
+        revision_uuid = await get_mapped_uuid("app_variant_revisions", revision_id)
+        associated_variants.append(
+            {
+                "human_evaluation_id": evaluation_id,
+                "variant_id": variant_uuid,
+                "variant_revision_id": revision_uuid,
+            }
+        )
+    return associated_variants
+
+
+async def transform_human_evaluation(evaluation):
+    app_uuid = await get_mapped_uuid("app_db", evaluation["app"].id)
+    user_uuid = await get_mapped_uuid("users", evaluation["user"].id)
+    test_set_uuid = await get_mapped_uuid("testsets", evaluation["testset"].id)
+    evaluation_uuid = generate_uuid()
+
+    await store_mapping("human_evaluations", evaluation["_id"], evaluation_uuid)
+
+    transformed_evaluation = {
+        "id": evaluation_uuid,
+        "app_id": app_uuid,
+        "user_id": user_uuid,
+        "status": evaluation["status"],
+        "evaluation_type": evaluation["evaluation_type"],
+        "testset_id": test_set_uuid,
+        "created_at": get_datetime(evaluation.get("created_at")),
+        "updated_at": get_datetime(evaluation.get("updated_at")),
+    }
+
+    associated_variants = await convert_human_evaluations_associated_variants(
+        evaluation["variants"], evaluation["variants_revisions"], evaluation_uuid
+    )
+
+    return transformed_evaluation, associated_variants
+
+
+async def transform_human_evaluation_scenario(scenario):
+    user_uuid = await get_mapped_uuid("users", scenario["user"].id)
+    evaluation_uuid = await get_mapped_uuid(
+        "human_evaluations", scenario["evaluation"].id
+    )
+    variant_uuid = str(await get_mapped_uuid("app_variants", scenario["vote"]))
```

Comment:
expect here the variant_uuid to be empty! There is always the case that the user has deleted the app_variant after creating the evaluation. This case needs to be handled correctly.